migtests/lib/yb.py

```diff
@@ -104,6 +104,11 @@ def get_distinct_values__of_column_of_table(self, table_name, column_name, schem
 		cur.execute(f"select distinct({column_name}) from {schema_name}.{table_name}")
 		return [value[0] for value in cur.fetchall()]
 
+	def get_all_values__of_column_of_table(self, table_name, column_name, schema_name="public") -> List[Any]:
+		cur = self.conn.cursor()
+		cur.execute(f"select {column_name} from {schema_name}.{table_name}")
+		return [value[0] for value in cur.fetchall()]
```

Comment:
Not worth a separate method. See if you can use a generic method like `run_query(query)`